pages/index.js

```diff
@@ -1,7 +1,106 @@
-export default function HomePage() {
-	return (
-		<>
-			<h1>My Cosmetics Store</h1>
-		</>
-	);
+import styled from "styled-components";
```

Comment:
```suggestion
import styled from "styled-components";
```
All the Imports should be placed at the top of the file